project_task_category/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Project Task Configurable Category
+==================================
+
+This module extends the functionality of Project to add a category on the task and support filter and group.
+
```

Comment:
The difference with tag is that this category is m2o and can be used for filtering and more important grouping or reporting.